go-controller/pkg/node/route_manager_test.go

```diff
@@ -34,7 +34,9 @@ var _ = ginkgo.Describe("Route Manager", func() {
 	loIP := net.IPv4(127, 1, 1, 1)
 	loIPDiff := net.IPv4(127, 1, 1, 2)
 	loGWIP := net.IPv4(127, 1, 1, 254)
-	if os.Getuid() != 0 {
+
+	if os.Getenv("NOROOT") == "TRUE" {
```

Comment:
++ @martinkennelly this should also fix the d/s merge unit test issues? wondering if we should give it a run to be sure..

Reply:
yes, i tested it [1] and it looked ok and it follows how we are doing currently detecting root
[1]  https://github.com/openshift/ovn-kubernetes/pull/1694